Clustering/Core/src/test/java/org/tribuo/neighbours/SGDVectorsFromCSV.java

```diff
@@ -0,0 +1,47 @@
+package org.tribuo.neighbours;
+
+import org.junit.jupiter.api.Test;
+import org.tribuo.math.la.DenseVector;
+import org.tribuo.math.la.SGDVector;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SGDVectorsFromCSV {
```

Comment:
Getting SGDVectors from csv's is straight-forward, but using the `GaussianClusterDataSource` seemed easier.